lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -58,7 +58,17 @@ sub index : Chained('root') PathPart('') Args(0) {
             = $c->model('API::Favorite')->by_user( $author->{user} )->recv;
         $took += $faves_data->{took} || 0;
 
-        $faves = [ map { $_->{fields} } @{ $faves_data->{hits}->{hits} } ];
+        my @all_fav = map { $_->{fields}->{distribution} }
+            @{ $faves_data->{hits}->{hits} };
+        my $noLatest = $c->model('API::Release')->no_latest(@all_fav)->recv;
```

Comment:
>The hash access will be slightly faster, so that's fine, but why not return a hash keyed on the  distribution name? I'm not sure what we win by checking $noLatest->{no_latest}->{$distro} rather than $no_latest->{$distro}

I don't know if it matters or not but I tried to mimic ES data structures (seen that all the other methods returns an hash with some metadata and results are nested under the "hits" key) because I thought that metadata is still important (see the took key):
```
{
	took      => $data->{took},
	no_latest => {
		map {
			my $distro = $_;
			( first { $_ eq $distro } @latest )
			? ()
			: ( $distro, 1 );
		} @distributions
	}
}
As bonus you can easily check if there are any hits at all by inspecting the no_latest hash's key number.
Maybe it could be renamed to "hits", so then you can access it with $noLatest->{hits}->{$distro} which is semantically better.